### ISSUE TITLE
git submodule update, unable to access

### ISSUE THREAD

**Issue by mark-walle** [aspyrx/setup-unix#1]
Moved the files to my home `/home/mwalle` (i assumed you meant all of the files from the entire repo), but when I try to run `git submodule update --init --recursive` i get `fatal: unable to access '/home/mwalle/.gitconfig': Input/output error`. same with `sudo`.

**Comment by aspyrx** [comment]
Hmm, that's weird. What operating system/environment is this? Did you move all the dotfiles (including `.git`, `.gitconfig`, etc.)?

**Comment by mark-walle** [comment]
Hi @aspyrx. This is on WSL ("Ubuntu for Windows 10"). I believe what happened is that I moved the contents with a wildcard something like `mv setup-utils/* ~` but failed to consider this wouldn't move the dotfiles. Retrying now, I found that I couldn't use a wildcard like `mv setup-unix/.[!.]* ~` because I already some files like `.local` in my home. I ended up using `rsync -av setup-unix/.[!.]* ~` which moves and merges everything, and that appears to have resolved the issue.
